lib/RdfXmlParser.ts

```diff
@@ -172,9 +172,9 @@ export class RdfXmlParser extends Transform {
   /**
    * Parses the given text stream into a quad stream.
    * @param {NodeJS.EventEmitter} stream A text stream.
-   * @return {NodeJS.EventEmitter} A quad stream.
+   * @return {RDF.Stream} A quad stream.
    */
-  public import(stream: EventEmitter): EventEmitter {
+  public import(stream: EventEmitter): RDF.Stream {
```

Comment:
Shouldn't this be the other way around? Input RDF.Stream and output EventEmitter?

Reply:
Nope. It's a parser so you parse a basic stream and produce a quad stream.

Reply:
Yes, of course...